packages/cli/src/util/deploy/create-git-meta.ts

```diff
@@ -0,0 +1,149 @@
+import fs from 'fs-extra';
+import { join } from 'path';
+import ini from 'ini';
+import git from 'git-last-commit';
+import {
+  BitbucketMeta,
+  GitHubMeta,
+  GitLabMeta,
+  GitMeta,
+  RepoData,
+} from '../../types';
+
+function getLastCommit(directory: string): Promise<git.Commit> {
+  return new Promise((resolve, reject) => {
+    git.getLastCommit(
+      (err, commit) => {
+        if (err) reject(err);
```

Comment:
If there's an error here, `reject` will be called, but so will `resolve`. Can we add a `return` to the error case?